wled00/FX.cpp

```diff
@@ -2016,7 +2016,7 @@ uint16_t mode_palette() {
         colorIndex = ((inputSize - 112) * colorIndex) / 16;
       }
       // Finally, shift the palette a bit.
-      const int paletteOffset = (!inputAnimateShift) ? (inputShift-128) : (((strip.now * ((inputShift >> 3) +1)) & 0xFFFF) >> 8);
+      const int paletteOffset = (!inputAnimateShift) ? (inputShift) : (((strip.now * ((inputShift >> 3) +1)) & 0xFFFF) >> 8);
```

Comment:
@TripleWhy FYI I stepped down as a maintainer & developer of WLED and my decisions (past and present) may be overruled.

The original reasoning I had was that for existing/old presets (which lack new effect sliders/options) the default value for sliders/options should not interfere/clash with effect display (i.e. if `c1` is unset in preset it's value should be 128 and not 0 which would be similar as if `sx` is missing).

Looking at the bigger picture (and certain PRs in the queue) quite a few effects are ripe for consolidation and pruning. This will inevitably *break compatibility* so no matter how you look at it some users will be upset.
My 2c.